src/Selenium2Driver.php

```diff
@@ -1120,4 +1132,63 @@ private function trigger($xpath, $event, $options = '{}')
         $script = 'Syn.trigger("' . $event . '", ' . $options . ', {{ELEMENT}})';
         $this->withSyn()->executeJsOnXpath($xpath, $script);
     }
+
+    /**
+     * Uploads a file to the Selenium instance.
+     *
+     * Note that uploading files is not part of the official WebDriver
+     * specification, but it is supported by Selenium.
+     * @see https://github.com/SeleniumHQ/selenium/blob/master/py/selenium/webdriver/remote/webelement.py#L533
```

Comment:
Please move `@see` tag after `@throws`. Or in any place, where other methods in this project has `@see` tag placed.